ansible-tests/validations/library/config_parser.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import ConfigParser
```

Comment:
ConfigParser is only for python 2, you'll have to either resort to using six of find another method to load the password.

Since we're considering switching to ansible 2, the built-in [ini lookup](http://docs.ansible.com/ansible/playbooks_lookups.html) could be an option.